TundraTD/Assets/Scripts/Spells/SpellAttribute.cs

```diff
@@ -12,14 +12,17 @@ public class SpellAttribute : Attribute
         /// A core element used to create a spell.
```

Comment:
We don't have to put so many xml docs in this file. Probably more descriptive names